astropy/tests/pytest_plugins.py

```diff
@@ -43,8 +44,13 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption("--remote-data", action="store_true",
+
+    # The following means that if --remote-data is not specified, the default
+    # is 'astropy', but if it is specified without arguments, it defaults to
```

Comment:
Ah but if you do ``--remote-data`` on its own, it does the old behavior and defaults to ``any``